<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HalconWidgetClass</class>
 <widget class="QMainWindow" name="HalconWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HalconWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="layout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>292</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>图像操作</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="readImageBtn">
       <property name="text">
        <string>读取图像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="drawCircleBtn">
       <property name="text">
        <string>绘制圆形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="drawRectangleBtn">
       <property name="text">
        <string>绘制矩形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="drawRotateBtn">
       <property name="text">
        <string>绘制旋转矩形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="drawEllipseBtn">
       <property name="text">
        <string>绘制椭圆</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearROIBtn">
       <property name="text">
        <string>清除ROI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetImageBtn">
       <property name="text">
        <string>恢复原图</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>370</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HalconWidget.qrc"/>
 </resources>
 <connections/>
</ui>
